.github/workflows/testing_apps.yml

```diff
@@ -0,0 +1,48 @@
+name: Testing apps
```

Comment:
Is there a more descriptive name we could use here?

Reply:
I don't know, this workflow means to test the installation and run of Encore on _real_ projects. 

Do you have something in mind?